tests/phpunit/CRM/Core/BAO/CustomQueryTest.php

```diff
@@ -49,8 +52,8 @@ public function testSearchCustomDataDateRelative() {
       "civicrm_value_testsearchcus_1.date_field_2 BETWEEN '" . date('Y') . "0101000000' AND '" . date('Y') . "1231235959'",
       $queryObj->_where[0][0]
     );
-    $this->assertEquals("date field is This calendar year (between January 1st, " . date('Y') . " 12:00 AM and December 31st, " . date('Y') . " 11:59 PM)", $queryObj->_qill[0][0]);
-    $queryObj = new CRM_Core_BAO_CustomQuery($params);
+    $this->assertEquals('date field is This calendar year (between January 1st, ' . date('Y') . " 12:00 AM and December 31st, " . date('Y') . " 11:59 PM)", $queryObj->_qill[0][0]);
+    $queryObj = new CRM_Contact_BAO_Query($params);
```

Comment:
@eileenmcnaughton i wonder about this does changing this have any meaningful impact on what we are testing?

Reply:
I think in a good way - comments on the commit https://github.com/civicrm/civicrm-core/pull/15719/commits/2605f837ef3f20ac7e36bd1d8dbc8fc0e9b6b3c7